modules/ztp-image-based-upgrade-creating-backup-resources.adoc

```diff
@@ -165,7 +167,38 @@ $ oc apply -f oadp-cm-example.yaml
 
 .. If you are using GitOps ZTP, push the generated `ConfigMap` object to your git repository.
 
-. (Optional) To keep your custom catalog sources after the upgrade, add them to the `spec.extraManifest` in the `ImageBasedUpgrade` CR. For more information, see xref:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/operators/index#olm-catalogsource_olm-understanding-olm[Catalog source].
+. (Optional) To keep your custom catalog sources after the upgrade, generate another `ConfigMap` object for your catalog sources and add them to the `spec.extraManifest` in the `ImageBasedUpgrade` CR. For more information, see xref:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.15/html-single/operators/index#olm-catalogsource_olm-understanding-olm[Catalog source].
```

Comment:
For more information IMHO should be enhanced to 

For more information about catalog sources   (otherwise one may assume it's about extra manifests